lucille-core/src/test/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnectorTest.java

```diff
@@ -86,10 +89,16 @@ public void testDatabaseConnector() throws Exception {
     assertEquals("Blaze", docsSentForProcessing.get(2).getStringList("name").get(0));
     assertEquals("Cat", docsSentForProcessing.get(2).getStringList("type").get(0));
     
+    connector.close();

```

Comment:
Should call assertEquals(2, dbHelper.checkNumConnections()) or something similar, right before connector close. Otherwise wise we're asserting 1 connection at the beginning and 1 at the end but never seeing it change anywhere.